src/pseudopeople/configuration/noise_configuration.py

```diff
@@ -70,40 +72,93 @@ def get_value(
                 f"Available column noise types are {COLUMN_NOISE_TYPES}."
             )
         # get value
-        parameter_tree: LayeredConfigTree = config[noise_type]
+        parameter_tree: LayeredConfigTree = config.get_tree(noise_type)
         if parameter_name not in parameter_tree:
             raise ValueError(
                 f"The parameter {parameter_name} was not found for {noise_type} in the configuration. "
                 f"Available parameters are {list(parameter_tree.keys())}."
             )
-        noise_value: int | float | LayeredConfigTree = parameter_tree[parameter_name]
+        noise_value: int | float | LayeredConfigTree = parameter_tree.get(parameter_name)
         converted_noise_value: int | float | dict = (
-            noise_value.to_dict()
+            # not sure how to tell mypy the types in this dict
+            noise_value.to_dict()  # type: ignore [assignment]
```

Comment:
I'd like to hear a bit more about why we need to ignore this

Reply:
No longer necessary